CakePHP/Tests/Commenting/FunctionCommentUnitTest.inc

```diff
@@ -1,12 +1,41 @@
 <?php
-
-namespace CakePHP;
+namespace Beakman;
 
 use Other\Crap;
 use Other\Error as OtherError;
 
-class Throws
+class Foo
```

Comment:
@markstory that's the way of unit testing sniff since a while now, we were just not using it. Also you can use `.inc.fixed` to check for the fixer result but I didn't just yet. 
I've been watching how it was done in the [Other standard](https://github.com/squizlabs/PHP_CodeSniffer/tree/3.0/src/Standards/Generic/Tests/Arrays)